lib/workers/repository/update/pr/index.ts

```diff
@@ -379,6 +404,36 @@ export async function ensurePr(
         );
         updatePrConfig.targetBranch = config.baseBranch;
       }
+
+      if (labelsNeedUpdate) {
+        logger.debug(
+          {
+            branchName,
+            prCurrentLabels,
+            configuredLabels,
+          },
+          'PR labels have changed',
+        );
+
+        // Divide labels into three categories:
+        // i) addLabels: Labels that need to be added
+        // ii) removeLabels: Labels that need to be removed
+        // iii) labels: New labels for the PR, replacing the old labels array entirely.
+        // This distinction is necessary because different platforms update labels differently
+        // For more details, refer to the updatePr function of each platform.
+
+        const [addLabels, removeLabels] = getChangedLabels(
+          prCurrentLabels,
+          configuredLabels,
+        );
+
+        // for Gitea
+        updatePrConfig.labels = configuredLabels;
+
+        // for Github, Gitlab
```

Comment:
```suggestion
        // for GitHub, GitLab
```

We should follow the brand's capitalization conventions. :wink: